inst/private/store_vars_in_python.m

```diff
@@ -25,6 +26,8 @@
       var_pyobj.append (pyeval (sympy (x)))
     elseif (iscell (x))
       var_pyobj.append (store_vars_in_python (x))
+    elseif (isequal (x, []))
+      var_pyobj.append (py.list ());
```

Comment:
I realize this 2 lines of code causes a regression because `isequal ('', [])` evaluates to true which is highly surprising to me. Is this for compatability or is it a bug?

Reply:
I don't know but I suspect its for compatibility.  I suggest you ask on discourse.

Good catch!  File a new issue for that and milestone it for 3.0.0 please!